mteb/models/__init__.py

```diff
@@ -160,7 +161,7 @@ def model_meta_from_sentence_transformers(model: SentenceTransformer) -> ModelMe
     sentence_transformers_models,
     vista_models,
     voyage_models,
-    google_models,
```

Comment:
was this accidentally removed btw?

Reply:
The item is duplicated from line 146/147.